src/ModalExample.elm

```diff
@@ -0,0 +1,116 @@
+module ModalExample exposing (..)
```

Comment:
I don't think we want this file either, do we...?

Reply:
not sure what's wrong! git kept re-adding that file. should be fixed now.